.github/workflows/build.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           set -o pipefail
           curl https://raw.githubusercontent.com/Kha/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain none -y
           ~/.elan/bin/lean --version
-          echo "::add-path::$HOME/.elan/bin"
-          echo "::set-env name=short_lean_version::$(~/.elan/bin/lean --run scripts/lean_version.lean)"
+          echo "$HOME/.elan/bin" >> $GITHUB_PATH
```

Comment:
Not unhappy with the change, but it's completely unrelated, right?

Reply:
Yes, Bryan mentioned a link to a GH blog post about some vulnerability that is fixed by this change.